lua/glance/init.lua

```diff
@@ -392,7 +392,10 @@ Glance.actions = {
         })
       end
     end
-    vim.fn.setqflist(qf_items, 'r')
+    vim.fn.setqflist(qf_items, ' ')
+    vim.fn.setqflist({}, 'a', {
+      title = 'Glance: ' .. glance.list.winbar.last_values.title,
+    })
```

Comment:
Annoyingly, you can't set a title when you set the list items, so the second call uses the append flag to set the title on the list we just added. See `:h setqflist-what`

Reply:
We can set the items with the title:
```lua
vim.fn.setqflist({}, ' ', {
  items = qf_items,
  nr = '$',
  title = 'Glance: ' .. glance.list.winbar.last_values.title,
})
```

```md
If {action} is not present or is set to ' ', then a new list is created.
To add a new quickfix list at the end of the stack, set "nr" in {what} to "$".

Reply:
Let's not use `glance.list.winbar` for the title. Winbar can be disabled by the user which will cause error here.
For now let's keep it just 'Glance', i'll have to rearrange some code to properly retrieve current LSP method name.

Reply:
Done!